docker/DockerfileNoetic

```diff
@@ -0,0 +1,77 @@
+# Start with ROS base image
+FROM ros:noetic-ros-base
+
+# Tools useful during development.
+RUN apt update \
+ && apt install -y --no-install-recommends\
+        cppcheck \
+        curl \
+        cmake \
+        lsb-release \
+        gdb \
+        git \
+        python3-dbg \
+        python3-pip \
+        python3-venv \
+        ruby \
+        software-properties-common \
+        sudo \
+        wget \
+        libeigen3-dev \
+        pkg-config \
+        protobuf-compiler \
+ && apt clean
+
+# Set distributions
+ARG DIST=noetic
+ARG GAZ=gazebo11
+
+
+# setup sources.list
+# ROS
+RUN echo "deb http://packages.ros.org/ros/ubuntu focal main" > /etc/apt/sources.list.d/ros1-latest.list
+# Gazebo
+RUN . /etc/os-release \
+    && echo "deb http://packages.osrfoundation.org/gazebo/$ID-stable `lsb_release -sc` main" > /etc/apt/sources.list.d/gazebo-latest.list
+
+
+# setup keys
+# ROS
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
+# Gazebo
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
```

Comment:
I think you might be able to go without this key. Going by [this tutorial](http://gazebosim.org/tutorials?tut=install_ubuntu), you shouldn't need it. I don't have this key in my Dockerfile. But somebody else should test it.

Reply:
https://github.com/osrf/docker_images/issues/535#issuecomment-851505821

Reply:
Oh, I was referring to the `D2486...` one – I think that one might not be needed.
The expired key is the `C1CF...` one which has been extended.

Reply:
Just a single data point, but I was unable to get this to run without both keys.